types/vinyl-file/tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
     "compilerOptions": {
+        "esModuleInterop": true,
```

Comment:
Unless vinyl and vinyl-file both require esModuleInterop to compile, this should stay `false` so that people who don't use `tsc --init` won't be inadvertently broken by changes.

Reply:
@sandersn sadly, without `esModuleInterop`, my test which should proof, that the previous type declaration did not work in projects with `esModuleInterop` enabled, doesn't work.

Is there some way to enable `esModuleInterop` for the test-file only?